app/hooks/hooks.types.ts

```diff
@@ -1,53 +1,7 @@
 import { Price } from "@app/components/price-graph"
-import { WalletCurrency } from "@app/types/amounts"
 import { INetwork } from "@app/types/network"
 import { GaloyGQL } from "@galoymoney/client"
 
-type mobileVersions = {
-  currentSupported: string
-  minSupported: string
-  platform: string
-}
-
-type Wallet = {
-  readonly __typename: "BTCWallet" | "USDTWallet"
-  readonly id: string
-  readonly balance: number
-  readonly walletCurrency: WalletCurrency
-}
-
-type UserTransactions = {
-  readonly __typename: "TransactionConnection"
-  edges: [
-    {
-      readonly __typename: "TransactionEdge"
-      readonly node: object[]
-    },
-  ]
-  pageInfo: {
-    hasNextPage: boolean
-    hasPreviousPage: boolean
-    startCursor: string
-    endCursor: string
-  }
-}
-
-type User = {
-  readonly __typename: "User"
-  defaultAccount: {
-    readonly __typename: "ConsumerAccount"
-    readonly defaultWalletId: string
-    readonly id: string
-    readonly transactions: UserTransactions
-    readonly wallets: Wallet[]
-  }
-  readonly id: string
-  readonly language: string
-  readonly phone: string
-  readonly quizQuestions: object[]
-  readonly username: string
-}
-
 export type useMainQueryOutput = {
```

Comment:
the whole query response is given to you fully typed by codegen. you don't need to redefine it here.

Reply:
Yeah. I have removed the types I manually added. These types are needed for the useMainQuery output because we modify the data response in the hook here https://github.com/GaloyMoney/galoy-mobile/blob/4705306b1e9f24ae67c00506b610f358031a2e39/app/hooks/use-main-query.ts#L51